splash/qtutils.py

```diff
@@ -299,3 +310,42 @@ def qt_header_items(request_or_reply):
         (name, request_or_reply.rawHeader(name))
         for name in request_or_reply.rawHeaderList()
     ]
+
+
+def qt_send_key(key, target):
+    """
+    Send a key event that might be defined using emacs keyboard macro syntax
+    to target qt object. For example:
+        - <Space> -> Qt.Key_Space
+        - <FooBar> -> Qt.Key_FooBar
+    See: http://doc.qt.io/qt-5/qt.html#Key-enum for valid key types
+    """
+
+    # Try to match "<Key_Name>"
+    key_match = re.match(r'^<(\w+)\>$', key)
+    if key_match:
+        key_type = getattr(Qt, 'Key_%s' % key_match.group(1), None)
+        if not key_type:
+            raise TypeError('Unknown key: %s' % key_match.group(1))
```

Comment:
TypeError is used when type of Python variable is incorrect, e.g. when int is passed instead of string. I think it should be ValueError.